Kudu.Contracts/Settings/DeploymentSettingsExtension.cs

```diff
@@ -211,9 +211,10 @@ private static TimeSpan GetTimeSpan(IDeploymentSettingsManager settings, string
             return defaultValue;
         }
 
-        public static string GetSiteExtensionRemoteUrl(this IDeploymentSettingsManager settings)
+        public static string GetSiteExtensionRemoteUrl(this IDeploymentSettingsManager settings, out bool isDefault)
         {
             string value = settings.GetValue(SettingsKeys.SiteExtensionsFeedUrl);
+            isDefault = String.IsNullOrEmpty(value);
             return !String.IsNullOrEmpty(value) ? value : DefaultSiteExtensionFeedUrl;
```

Comment:
Instead of calling ```String.IsNullOrEmpty(value)``` twice, why not just use ```!isDefault``` on the line with the ternary operator?

Reply:
Good point, I'll change that